packages/webchat/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  // Manage references here
+  "references": [
+    {
+      "path": "../socket"
+    },
+    {
+      "path": "../components"
+    }
+  ],
+
+  // Defaults (don't change this)
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/.tsbuildinfo",
+
+    // Specific settings for npm package
+    "target": "es6",
+    "declaration": true,
+    "sourceMap": false,
+
+    // Because original webchat was written with decorators
+    "experimentalDecorators": true,
+    // We use react
+    "jsx": "react"
+  },
+  "exclude": ["dist", "node_modules"]
+}
```

Comment:
Here the tsconfig file is pretty basic as well. It gets adjusted in the next PR to compile correctly.